plugins/git/git-worktree.nix

```diff
@@ -71,13 +74,10 @@ in
 
       extraPackages = [ pkgs.git ];
 
-      extraConfigLua =
-        let
-          telescopeCfg = ''require("telescope").load_extension("git_worktree")'';
-        in
-        ''
-          require('git-worktree').setup(${helpers.toLuaObject setupOptions})
-          ${if cfg.enableTelescope then telescopeCfg else ""}
```

Comment:
I assume this wasn't done to ensure correct setup order? If so this PR may break that.